scripts/utils/constructProposalCalldata.ts

```diff
@@ -47,3 +58,38 @@ export async function constructProposalCalldata(proposalName: string): Promise<s
 
   return calldata;
 }
+
+function getTimelockCalldata(proposal: ExtendedAlphaProposal, proposalInfo: ProposalDescription): string {
+  const proposeFuncFrag = new Interface([
+    'function scheduleBatch(address[] calldata targets,uint256[] calldata values,bytes[] calldata data,bytes32 predecessor,bytes32 salt,uint256 delay) public',
+    'function executeBatch(address[] calldata targets,uint256[] calldata values,bytes[] calldata data,bytes32 predecessor,bytes32 salt) public'
+  ]);
+
+  const combinedCalldatas = [];
+  for (let i = 0; i < proposal.targets.length; i++) {
+    const sighash = utils.id(proposal.signatures[i]).slice(0, 10);
+    combinedCalldatas.push(`${sighash}${proposal.calldatas[i].slice(2)}`);
```

Comment:
doesn't this assume that the calldatas are already fully expanded from the file?

Reply:
this is the exact same code path to generate calldata for a DAO proposal, just uses the inputs in a different abi